config/jobs/image-pushing/releng/k8s-staging-artifact-promoter.yaml

```diff
@@ -4,7 +4,7 @@ postsubmits:
       cluster: k8s-infra-prow-build-trusted
       annotations:
         testgrid-dashboards: sig-release-releng-blocking, sig-release-master-informing, sig-release-image-pushes, sig-k8s-infra-gcb
```

Comment:
this should probably be a follow-up, but I'm not convinced this should be in relaese-master-informing? it's not testing kubernetes. (vs sig-release-releng-blocking, that makes sense)

Reply:
I'll open a thread in #sig-release